hooks/terraform_docs.sh

```diff
@@ -147,14 +162,16 @@ function terraform_docs {
         ;;
       --use-standard-markers)
         use_standard_markers=$value
+        common::colorify "yellow" "WARNING: --use-standard-markers is deprecated and will be removed in the future."
+        common::colorify "yellow" "         All needed changes already done by the hook, feel free to remove this line."
         ;;
     esac
   done
 
-  if [ "$use_standard_markers" = true ]; then
-    # update the insertion markers to those used by terraform-docs
-    insertion_marker_begin="$standard_insertion_marker_begin"
-    insertion_marker_end="$standard_insertion_marker_end"
+  if [ "$use_standard_markers" = false ]; then
```

Comment:
Using Bashism (as we relay upon Bash) may be worthwhile (I also prefer `==` over `=` to not mess with assignment operator):
```suggestion
  if [[ $use_standard_markers == false ]]; then
```